otherlibs/dune-rpc/private/types.ml

```diff
@@ -27,7 +27,7 @@ end
 module Version = struct
   type t = int * int
 
-  let latest = 3, 11
+  let latest = 3, 12
```

Comment:
not sure whether this should be in this PR

Reply:
a separate PR would be best.